docs/operating-procedures.md

```diff
@@ -94,11 +94,6 @@ possible to create a CSV output of the results:
 
     $ screen -dmS cache parallel --eta --sshloginfile nodes -a ids.txt --results /tmp/cache/ --sqlandworker csv:////%2Ftmp%2Fcache.csv -j10 '/opt/omero/server/OMERO.server/bin/omero render -s localhost -u public -w public test --force'
 
-## Analysis IDR
```

Comment:
technically speaking it is. 
Craig forked the repo and he is using it for the new infra 
cf. https://github.com/bioimagearchive/k8s-jupyterhub

Reply:
We can remove it from our doc but maybe add a link to new infra

Reply:
@jburel : where do you want those docs? MT wiki? Or something more public?